botx/models/chats.py

```diff
@@ -28,3 +29,47 @@ class ChatFromSearch(BotXBaseModel):
 
     #: users in chat.
     members: List[UserFromChatSearch]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+
+class BotChat(BotXBaseModel):
+    """Chat from list."""
+
+    #: name of chat.
+    name: str
+
+    #: description of chat
+    description: Optional[str]
+
+    #: type of chat.
+    chat_type: ChatTypes
+
+    #: ID of chat.
+    group_chat_id: UUID
+
+    #: users in chat.
+    members: List[UUID]
+
+    #: creation datetime of chat.
+    inserted_at: datetime
+
+    #: update datetime of chat.
+    updated_at: datetime
+
+
+class BotChatList(BotXBaseModel):
```

Comment:
Why not `List[ChatFromList]`?

Reply:
To show explicit that `get_bot_chats` method returning **List** of bot's chats I must create "Custom root type" (https://github.com/samuelcolvin/pydantic/issues/675#issuecomment-513029543)

I can't just return `List[BotChat]`, code will brake [here](https://github.com/ExpressApp/pybotx/blob/a6de96f46bccb878f5f1d3d28e531d6bce2333e1/botx/clients/methods/v3/chats/list.py#L9)